samples/python/multiview_calibration.py

```diff
@@ -0,0 +1,220 @@
+    ax.view_init(azim=90, elev=-40)
+
+def plotProjection(points_2d, pattern_points, rvec0, tvec0, rvec1, tvec1, K, dist_coeff, is_fisheye, image_size, image=None):
+    rvec2, tvec2 = cv.composeRT(rvec0, tvec0, rvec1, tvec1)[:2]
+    if is_fisheye:
+        points_2d_est = cv.fisheye.projectPoints(pattern_points, rvec2, tvec2, K, dist_coeff.flatten())[0]
+    else:
+        points_2d_est = cv.projectPoints(pattern_points, rvec2, tvec2, K, dist_coeff)[0].reshape(-1,2)
+
+    fig = plt.figure()
+    if image is None:
+        ax = fig.add_subplot(111)
+        ax.scatter(points_2d[:,0], points_2d[:,1], color='red', marker='.')
+        ax.scatter(points_2d_est[:,0], points_2d_est[:,1], color='green', marker='+')
+        for pt1, pt2 in zip(points_2d, points_2d_est):
+            ax.plot([pt1[0], pt2[0]], [pt1[1], pt2[1]], color='blue')
+        dim_box = getDimBox(points_2d)
+        ax.set_aspect('equal', 'box')
+        # ax.set_xlim(0, image_size[0]); ax.set_ylim(0, image_size[1])
+        ax.set_xlabel('x', fontsize=20); ax.set_ylabel('y', fontsize=20)
+        ax.legend(['given', 'back-projected'], fontsize=15)
+        ax.set_title('Comparison of given points and back-projected', fontsize=17)
+    else:
+        circle_sz = 5
+        for pt1, pt2, in zip(points_2d, points_2d_est):
+            cv.circle(image, np.array(pt1, dtype=int), circle_sz, (0, 0, 255), -1)
+            cv.circle(image, np.array(pt2, dtype=int), circle_sz, (0, 255, 0), -1)
+            cv.line(image, np.array(pt1, dtype=int), np.array(pt2, dtype=int), (255, 0, 0), 1)
+        plt.imshow(image)
+
+def calibrateFromPoints(pattern_points, image_points, image_sizes, is_fisheye, image_names=None):
+    """
+    pattern_points: NUM_POINTS x 3 (numpy array)
+    image_points: NUM_CAMERAS x NUM_FRAMES x NUM_POINTS x 2
+    is_fisheye: NUM_CAMERAS (bool)
+    image_sizes: NUMCAMERAS x [width, height]
+    """
+
+    num_cameras = len(image_points)
+    num_frames = len(image_points[0])
+    visibility = np.zeros((num_cameras, num_frames), dtype=int)
+    pattern_points_all = [pattern_points for i in range(num_frames)]
+    # pattern_points_all = [pattern_points] * num_frames
+    for i in range(num_cameras):
+        for j in range(num_frames):
+            visibility[i,j] = int(len(image_points[i][j]) != 0)
+
+    success, rvecs, Ts, Ks, distortions, rvecs0, tvecs0, errors_per_frame, output_pairs = \
+        cv.calibrateMultiview(objPoints=pattern_points_all,
+                              imagePoints=image_points,
+                              imageSize=image_sizes,
+                              visibility=visibility,
+                              #Ks=Ks,
+                              #distortions=distortions,
+                              is_fisheye=np.array(is_fisheye, dtype=int),
+                              USE_INTRINSICS_GUESS=False,
+                              flags_intrinsics=0)
```

Comment:
Current version for 5.x fails with exception:
```
Traceback (most recent call last):
  File "/home/alexander/Projects/OpenCV/opencv-next/samples/python/multiview_calibration.py", line 220, in <module>
    params.pattern_type, [bool(int(v)) for v in params.fisheye.split(',')], params.pattern_distance)
  File "/home/alexander/Projects/OpenCV/opencv-next/samples/python/multiview_calibration.py", line 194, in calibrateFromImages
    calibrateFromPoints(pattern, image_points_cameras, image_sizes, is_fisheye, all_images_names)
  File "/home/alexander/Projects/OpenCV/opencv-next/samples/python/multiview_calibration.py", line 93, in calibrateFromPoints
    flags_intrinsics=0)
cv2.error: OpenCV(5.0.0-pre) :-1: error: (-5:Bad argument) in function 'calibrateMultiview'
> Overload resolution failed:
>  - Required argument 'Ks' (pos 5) not found
>  - Required argument 'Ks' (pos 5) not found
```

Reply:
fixed, I forgot to add CV_OUT.

Reply:
> @ivashmak Please create PR with test data to opencv_extra with the same branch name. XMLs with board points are missing.

Here is PR: https://github.com/opencv/opencv_extra/pull/1001